repository-springboot/pom.xml

```diff
@@ -172,6 +172,22 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>com.heroku.sdk</groupId>
+        <artifactId>heroku-maven-plugin</artifactId>
+        <version>2.0.3</version>
+        <configuration>
+          <appName>${artifactId}</appName>
+          <includeTarget>false</includeTarget>
+          <includes>
+            <include>${project.build.directory}/${project.build.finalName}.jar</include>
+          </includes>
+          <jdkVersion>${java.version}</jdkVersion>
+          <processTypes>
+            <web>java $JAVA_OPTS -jar target/${project.build.finalName}.jar</web>
```

Comment:
You have specified `<includeTarget>false</includeTarget>` so is `target` needed in the command? I haven't tried it, but it shouldn't work I guess if target is not included.

Reply:
This is the configuration when wanting to deploy only the executable jar which is what we wanted to do here.

Reply:
so it will actually upload it to heroku with target/<springboot>.jar directory structure? Then OK.